module.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux
```

Comment:
added the linux build tag to prevent linters from removing imports when they feel like it, and the module is intended to only support linux aarch64